src/viam/components/camera/camera.py

```diff
@@ -74,10 +77,10 @@ async def get_images(self, *, timeout: Optional[float] = None, **kwargs) -> Tupl
 
         ::
 
-            my_camera = Camera.from_robot(robot=robot, name="my_camera")
+            my_camera = Camera.from_robot(robot=machine, name="my_camera")
 
             images, metadata = await my_camera.get_images()
-            img0 = images[0].image
+            image_list = images[0]
```

Comment:
Technically this should be `image_list = images`. The `[0]` means it's only the first image.

Reply:
I think 0 is the list of images and 1 is the metadata, according to the docs?

Reply:
My impression was that the first image would be images[0][0]?

Reply:
If you look above, it's already split out. `images, metadata = await get_images...`, so it's already done the first [0].

Reply:
```suggestion
            first_image = images[0]
```

Reply:
oOoh ok ty